### PR TITLE
+slf #23409 support org.slf4j.Marker

### PR DESCRIPTION
**Pull request by ktoso**
Resolves https://github.com/akka/akka/issues/23409

No touching of akka-actor was needed thankfully and it's compatible 😎 

Missing docs, I'll add them after dinner

### CODE REVIEW

File: akka-slf4j/src/main/scala/akka/event/slf4j/Slf4jLogger.scala [hunk 1]
```diff
@@ -141,3 +142,16 @@ class Slf4jLoggingFilter(settings: ActorSystem.Settings, eventStream: EventStrea
...
+final class Slf4jLogMarker(marker: org.slf4j.Marker) extends LogMarker(name = marker.getName) {
+  /** Java API: get underlying [[org.slf4j.Marker]] */
+  def getMarker = marker
+}
+/** Factory for creating [[LogMarker]] that wraps [[org.slf4j.Marker]] */
+object Slf4jLogMarker {
+  def apply(marker: org.slf4j.Marker): Slf4jLogMarker = new Slf4jLogMarker(marker)
+
+  /** Java API */
+  def create(name: org.slf4j.Marker): Slf4jLogMarker = apply(name)
```
Comment (jrudolph) [inline akka-slf4j/src/main/scala/akka/event/slf4j/Slf4jLogger.scala]: Change parameter name to `marker` here as well.

**Review by jrudolph — APPROVED** [review]
LGTM, nice small implementation.

A small nitpick below.

**Comment by ktoso** [comment]
Thanks for review, fixed!

File: akka-slf4j/src/main/scala/akka/event/slf4j/Slf4jLogger.scala [hunk 2]
```diff
@@ -141,3 +142,16 @@ class Slf4jLoggingFilter(settings: ActorSystem.Settings, eventStream: EventStrea
   def isDebugEnabled(logClass: Class[_], logSource: String) =
     (eventStream.logLevel >= DebugLevel) && Logger(logClass, logSource).isDebugEnabled
 }
+
+/** Wraps [[org.slf4j.Marker]] */
+final class Slf4jLogMarker(marker: org.slf4j.Marker) extends LogMarker(name = marker.getName) {
```
Comment (wsargent) [inline akka-slf4j/src/main/scala/akka/event/slf4j/Slf4jLogger.scala]: I see how Java API can get at underlying `org.slf4j.Marker`, but don't see how Scala API gets at it -- there is no `val marker` and it's not a case class, so `marker` is not accessible.

Comment (ktoso) [inline akka-slf4j/src/main/scala/akka/event/slf4j/Slf4jLogger.scala]: good point, this marker should be a val, though the usage of getting it is pretty much the exact same thing among the APIs, perhaps I should drop the "java api" and just make marker a val and be done with it :)

**Review by johanandren — APPROVED** [review]
LGTM

**Comment by raboof** [comment]
cool

**Pull request merged**